server/lib/nl_detection.py

```diff
@@ -58,8 +76,61 @@ class ClassificationAttributes(ABC):
 @dataclass
 class SimpleClassificationAttributes(ClassificationAttributes):
   """Simple classification attributes."""
-  place_detected: Place
-  sv_detected: SV
+  pass
+
+@dataclass
+class RankingClassificationAttributes(ClassificationAttributes):
+  """Ranking classification attributes."""
+  # ranking_high is for queries like:
+  # "most populous cities ..."
+  # "top five best places ..."
+  # "highest amount of in ..."
+  ranking_high: bool
```

Comment:
make ranking_high and ranking_low enum as well?

Reply:
Good point, done!